src/modules/UI/scenes/ChangeMiningFee/style.js

```diff
@@ -4,31 +4,29 @@ import {StyleSheet} from 'react-native'
 import THEME from '../../../../theme/variables/airbitz'
 
 const styles = {
-  gradient: {
-    height: THEME.SPACER.HEADER
-  },
-  container: {
-    flexDirection: 'column'
-  },
-  headerContainer: {
-    padding: 25,
-    paddingBottom: 10
+
+  scene: {
+    flex: 1,
+    backgroundColor: THEME.COLORS.WHITE
   },
-  header: {
-    fontSize: 15,
-    color: THEME.COLORS.GRAY_1
+  gradient: {
+    height: 66,
```

Comment:
Have you checked that this works on android?
We have `THEME.SPACER.HEADER` specifically for this reason

Reply:
I haven't tested it on android.
Since @kylanhurt made that change he might have.
But anyway, THEME.SPACER.HEADER doesn't fit in this case so if we do have issues on Android, we will have to create a new style prop inside "theme" and do it a new PR since it's just a style issue.